OloEngine/src/OloEngine/Scene/SceneSerializer.cpp

```
@@ -200,6 +200,143 @@ namespace OloEngine
+            sanitizeFloat(se.InitialSize, 0.001f, 1.0f, 0.04f);
+            sanitizeFloat(se.SizeVariance, 0.0f, 0.5f, 0.02f);
+            sanitizeFloat(se.GravityScale, 0.0f, 5.0f, 0.3f);
+            sanitizeFloat(se.DragCoefficient, 0.0f, 20.0f, 2.0f);
+            sanitizeFloat(se.VelocityThreshold, 0.0f, 10.0f, 0.1f);
+            se.MaxParticles = std::clamp(se.MaxParticles, 256u, 65536u);
+        }
+    }
+
+    static void DeserializeSnowDeformerComponent(Entity& entity, const YAML::Node& node)
+    {
+        auto& sd = entity.AddComponent<SnowDeformerComponent>();
```

Clang (14.0.6)

[warning] variable name 'sd' is too short, expected at least 3 characters

(readability-identifier-length)